daq_main_common.py

```diff
@@ -40,8 +40,8 @@ def pybass_init():
 
   daq_utils.init_environment()
   daq_lib.init_var_channels()
-  #if getBlConfig(config_params.DETECTOR_OBJECT_TYPE) == config_params.DETECTOR_OBJECT_TYPE_LSDC:
-  det_lib.init_detector()  
+  if getBlConfig(config_params.DETECTOR_OBJECT_TYPE) != config_params.DETECTOR_OBJECT_TYPE_NO_INIT:
```

Comment:
not sure why NO_INIT is defined - can't you make use of one of the other states?